#include "variables"
# patch-info - enforce consistency rules for AUCTeX info files
#
# $Id: patch-info.in,v 1.2 2004/03/16 13:38:57 salve Exp $
#
<:=@COPYRIGHT:>//
#
--- doc/Makefile	2002-12-15 03:15:43.000000000 +0100
+++ doc/Makefile	2004-03-16 11:21:30.000000000 +0100
@@ -59,7 +59,7 @@
 
 install: auctex
 	-mkdir -p $(infodir)
-	cp auctex auctex-* $(infodir)
+	cp auctex.info* $(infodir)
 	@echo "**********************************************************"
 	@echo "** The \`auctex' info files are now installed."
 	@echo "** Edit \`$(infodir)/dir'"
--- doc/auc-tex.texi	2002-12-15 03:15:43.000000000 +0100
+++ doc/auc-tex.texi	2004-03-16 11:25:55.000000000 +0100
@@ -1,5 +1,5 @@
 \input texinfo
-@setfilename auctex
+@setfilename auctex.info
 @settitle AUC TeX
 @direntry
 * AUCTeX: (auctex).      A much enhanced LaTeX mode for GNU Emacs.
